extensions/machine-learning/src/views/models/manageModels/currentModelsComponent.ts

```diff
@@ -200,7 +202,6 @@ export class CurrentModelsComponent extends ModelViewBase implements IPageView {
 
 	private refreshComponents(): void {
 		if (this._formBuilder) {
```

Comment:
wont we have to still remove in the this._dataTable.isEmpty = true flow?

Reply:
remove is for cleaning up everything. no condition is needd